04-single-hop-6lowpan-icmp/test_spec04.py

```diff
@@ -286,3 +286,88 @@ def test_task10(riot_ctrl):
     time.sleep(60)
     assert pktbuf(pinged).is_empty()
     assert pktbuf(pinger).is_empty()
+
+
+@pytest.mark.iotlab_creds
+# nodes passed to riot_ctrl fixture
+@pytest.mark.parametrize('nodes',
+                         [pytest.param(['nrf52840dk', 'iotlab-m3',
+                                        'iotlab-m3'])],
+                         indirect=['nodes'])
+def test_task11(riot_ctrl):
+    nodes = (
+        riot_ctrl(0, APP, Shell, modules=["gnrc_pktbuf_cmd"]),
+        riot_ctrl(1, APP, Shell, modules=["gnrc_pktbuf_cmd"]),
+        riot_ctrl(2, APP, Shell, modules=["gnrc_pktbuf_cmd"]),
+    )
+
+    pinged = nodes[0]
+    pingers = nodes[1:]
+
+    pinged_netif, pinged_addr = lladdr(pinged.ifconfig_list())
+    pinged.ifconfig_set(pinged_netif, "channel", 26)
+    assert pinged_addr.startswith("fe80::")
+    for pinger in pingers:
+        pinger_netif, _ = lladdr(pinger.ifconfig_list())
+        pinger.ifconfig_set(pinger_netif, "channel", 26)
+
+    futures = []
+    for pinger in nodes[1:]:
+        out = pinger.ping6(pinged_addr,
+                           count=200, interval=0, packet_size=1232,
+                           async_=True)
+        futures.append(out)
+    wait_for_futures(futures)
+
+    time.sleep(60)
+    for node in nodes:
+        # add print to know which node's packet buffer is not empty on error
+        print("check pktbuf on", node.riotctrl.env.get("PORT"))
+        assert pktbuf(node).is_empty()
+
+
+@pytest.mark.local_only
```

Comment:
Why local only? Both boards are available at the `saclay` side (which is the default).